ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/platform.py

```diff
@@ -32,16 +36,16 @@
         },
     },
     {
-        "platforms": ["720XP", "722XP"],
-        "trident_forwarding_table_partition": "flexible exact-match 16384 l2-shared 98304 l3-shared 131072",
+        "platforms": ["720XP"],
+        "trident_forwarding_table_partition": "flexible exact-match 16384 l2-shared 32768 l3-shared 32768",
```

Comment:
Changing these defaults with no notice? Are the old values a bug or are you just taking the defaults on the platform? The old defaults came from a TOI somewhere.

Reply:
Saw your comment on the default values now. I think we should submit that as a separate fix so it will be clear in release notes.

Reply:
Ah yes that makes a lot of sense.  Will fix that now.